src/main/java/org/example/usersService/UserRoles.java

```diff
@@ -0,0 +1,18 @@
+package org.example.usersService;
+
+public class UserRoles extends User{
```

Comment:
you could have created an enum for user roles. We have a couple of predefined roles which makes it perfect for an enum.

Reply:
Why is UserRoles extending from User? Do we really need all of the logic from the User class in our UserRoles class?